package/cpp/MmkvLogger.h

```diff
@@ -11,25 +11,14 @@ class MmkvLogger {
 private:
   MmkvLogger() = delete;
 
-private:
-  template <typename... Args>
-  static std::string string_format(const std::string& format, Args... args) {
-    int size_s = std::snprintf(nullptr, 0, format.c_str(), args...) + 1; // Extra space for '\0'
-    if (size_s <= 0) {
-      throw std::runtime_error("Failed to format string!");
-    }
-    auto size = static_cast<size_t>(size_s);
-    std::unique_ptr<char[]> buf(new char[size]);
-    std::snprintf(buf.get(), size, format.c_str(), args...);
-    return std::string(buf.get(), buf.get() + size - 1); // We don't want the '\0' inside
-  }
 
 public:
-  static void log(const std::string& tag, const std::string& message);
+  static void log(const std::string& tag, const std::string& formatString, Args... args);
```

Comment:
This is not a template, you cannot use `Args...` here. This is why the C++ build is now failing.

Reply:
yes, I understand, but I ended up not using this project for my work, so I don't have time to fully flesh out this patch.  I was intending to simply illustrate some ways of avoiding the logging overhead and/or avoiding NSLog in release builds (which is discouraged)